File: MyBreak/DayThree/DayFour/Python/chapter4_task_three.py
account_number = int(input("Enter your account number: "))

if account_number < 10 or account_number > 10:
  print("Please enter a valid account number")
  
else:
  beginning_balance = int(input("Enter your balance at the beginning of the months: "))
  charges = int(input("Enter your charges"))
  credit_limit = int(input("Enter your credit limit applied to the customer this months: "))
  
  new_balance = beginning_balance + charges - credit_limit
  print("Your new balance is : " + str(new_balance))
  if new_balance < 0.0:
    print("You've exceeded your credit limit, please try again later.")
    

File: MyBreak/DayThree/DayFour/JavaScript/Chapter4TaskThree.js
const prompt = require("prompt-sync")()

function creditLimitCalculator(){
let newBalance = 0


let accountNumber = prompt("Enter your account number: ")
if(accountNumber > 10 || accountNumber < 10){
  console.log("Please enter a valid account number: ");
}
else{
let beginningBalance = prompt("Enter your balance at the beginning of a new month: ")
let charges = prompt("Enter the charges amount: ")
let creditLimit = prompt("Enter the total credit limit applied to the account this month: ")

newBalance = beginningBalance + charges - creditLimit
}
if(newBalance < 0.0){

  console.log("Credit limit exceeded")
}

console.log("Your new balance is " + newBalance)
}
creditLimitCalculator()